gn2/wqflask/views.py

```diff
@@ -1219,19 +1220,19 @@ def display_diffs_users():
                            files=files)
 
 
-@app.route("/genewiki/<symbol>")
-def display_generif_page(symbol):
+@app.route("/genewiki/<string:symbol>")
```

Comment:
what's your preference for the routes to use? The one I chose is `metadata/wiki/comment_id/edit`

Reply:
Let's leave this as is.